### ISSUE TITLE
[Accessibility: Timing] Warn user the session will expire in N seconds

### ISSUE THREAD

**Issue by alinefm** [kimchi-project/wok#77]
Before session expires, the user should be able to extend the session time for answer a question to keep logged.
